java/sage/Show.java

```diff
@@ -456,38 +426,30 @@ public boolean isIdentical(Show other)
   Show(DataInput in, byte ver, Map<Integer, Integer> idMap) throws IOException
   {
     super(in, ver, idMap);
-    Wizard wiz = Wizard.getInstance();
-    duration = in.readLong();
-    title = wiz.getTitleForID(readID(in, idMap));
 
-    // We lazily create these String to speed up loading time and reduce memory overhead
-    int size = in.readShort();
-    if (size == 0)
-      episodeNameStr = "";
-    else {
-      episodeNameBytes = new byte[size];
-      in.readFully(episodeNameBytes);
-    }
-    size = in.readShort();
-    if (size == 0)
-      descStr = "";
-    else {
-      descBytes = new byte[size];
-      in.readFully(descBytes);
-    }
+    int size        = 0;
```

Comment:
Please don't reorganize code that's not part of the change or modify formatting in code that's not part of the change...makes it harder to review and isolate faults that come from changes.